packages/typespec-python/src/lib.ts

```diff
@@ -14,6 +14,7 @@ export interface PythonEmitterOptions {
     "tracing"?: boolean;
     "company-name"?: string;
     "debug"?: boolean;
+    "flavor"?: "azure";
```

Comment:
here I hard coded the value to be lowercase. I thought it would be better to actually add the valid values as they came in instead of just typing this as a string